src/Microsoft.DotNet.InternalAbstractions/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for making code that uses file system and environment testable.",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I see. Those are good points. How about 1.0.0-preview2-1-*? Matching the rest of the Repo. Given that these are depending on non-release version of NuGet packages, I think it should be ok to call them preview and version them with the rest of the repo.